src/ipc/dbus/DBusIPCProxyBinder.cpp

```diff
@@ -171,6 +171,14 @@ void DBusIPCProxyBinder::checkRegistry()
     }
 }
 
+bool DBusIPCProxyBinder::isAllowedToConnect()
```

Comment:
"Allowed" suggests that there is some kind of permission somewhere, which is not the case.
I don't see the need to define a new method for this expression which is used only once anyway.